transports/webrtc/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "libp2p-webrtc"
-version = "0.9.0-alpha.1"
+version = "0.9.1-alpha"
```

Comment:
Why not `0.9.0-alpha.2`? We usually bump the numeric suffix if something is just a patch change. 
I know that the CI complained, but that might be more an issue of our CI script, rather than the version being actually invalid.

Reply:
I thought CI failure was an issue.  Reverted.

Reply:
This script currently doesn't handle pre-releases properly; it only compares major, minor and patch numbers. It's still on my TODO list to fix that, just didn't get to it yet. 
But in the mean time we shouldn't block on it.